ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -1546,6 +1546,9 @@ namespace
         case element::Type_t::f64:
             convert_v0::evaluate<element::Type_t::f64, OUT_ET>(op, outputs, inputs);
             break;
+        case element::Type_t::bf16:
```

Comment:
@ilyachur I've added this type to be able to run the unit test but this might also be useful for the fallback-to-ref-impl later on. Especially that ONNX opset 13 added support for bloat16 to most (or even all) ops.

Reply:
Hi @tomdol, I think we can add bf16 type. I don't have any objections.